comms/EclMeta.cpp

```diff
@@ -231,6 +259,68 @@ bool CEclMeta::LoadImports(const std::_tstring & path, const WPathVector & folde
     return found;
 }
 
+bool CEclMeta::LoadFunctions(const std::_tstring & path)
+{
+    bool found = false;
+
+    for (std::map<std::wstring, StlLinked<CEclMetaData> >::iterator itr = m_masterMeta.begin(); itr != m_masterMeta.end(); ++itr)
+    {
+        if (CEclFile *file = dynamic_cast<CEclFile*>(itr->second.get()))
+        {
+            found = file->GetFunctions(m_funcs) ? true : found;
```

Comment:
Should `break` at this point?

Reply:
Yes. Good catch!

Reply:
Actually, there should not be a break here. GetFunctions accumulates the functions. The found is just a general flag that something was found.

Reply:
Ok.